sql/src/main/java/org/apache/apex/malhar/sql/SQLExecEnvironment.java

```diff
@@ -0,0 +1,248 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.apex.malhar.sql;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+
+import org.apache.apex.malhar.sql.planner.RelNodeVisitor;
+import org.apache.apex.malhar.sql.schema.ApexSQLTable;
+import org.apache.apex.malhar.sql.table.Endpoint;
+import org.apache.calcite.config.Lex;
```

Comment:
I just went ahead with the checkstyle that is mentioned here: https://github.com/apache/apex-core/blob/master/codestyle-config/src/main/resources/apex_checks.xml This is what is used during build also for verification. That said, the travis and jenkins build are passing I believe the import order mentioned in this file is correct.

If we want to change import order in apex-core, that's a seperate discussion and outside the scope of this PR.